app/models/hotel.rb

```diff
@@ -0,0 +1,6 @@
+class Hotel < ApplicationRecord
+    #belongs_to :city
```

Comment:
Eh, the schema is going to have to change again once the City model is added. To keep things simple, I would advise waiting for City to be added. The design that we went over requires both models anyway.

Reply:
Cities table now merged